pulsar-client/src/test/java/org/apache/pulsar/client/impl/schema/AvroSchemaTest.java

```diff
@@ -423,4 +425,16 @@ public void testAvroSchemaUserDefinedReadAndWriter() {
         assertEquals(field1, foo.getField1());
     }
 
+    static class MyPojo {
+        public UUID uid;
+    }
+
+    @Test
+    public void testAvroUUID() {
+        org.apache.pulsar.client.api.Schema schema = org.apache.pulsar.client.api.Schema.AVRO(MyPojo.class);
+        MyPojo pojo = new MyPojo();
+        pojo.uid = UUID.randomUUID();
+        schema.encode(pojo);
```

Comment:
can we also `decode` ? and verify the result ?

Reply:
Added Schema.decode and test the result, it's ok.